.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+name: create release on new tags
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: [ '*.*.*' ]
```

Comment:
We'll want to make sure this works how we want. I think you mentioned you'd test this in a dev repo, I'd like to see that when you have it standing.

Reply:
I've changed it to something a little bit more robust (`[0-9]+.[0-9]+.[0-9]+`). Note that we don't prefix our version with `v`.

I've tested using a random string and the pipeline is not triggered.